grpc/src/main/java/com/linecorp/armeria/client/grpc/ArmeriaChannel.java

```diff
@@ -166,8 +166,8 @@ private DefaultClientRequestContext newContext(HttpMethod method, HttpRequest re
                 sessionProtocol,
                 options().requestIdGenerator().get(),
                 method,
-                uri().getRawPath(),
-                uri().getRawQuery(),
+                req.path(),
+                null,
```

Comment:
This fixes a bug in the gRPC client where a wrong path (usually `/`) is set to the context. /cc @anuraaga